source/draw/RasterYuvPlanar.ooc

```diff
@@ -67,12 +67,8 @@ RasterYuvPlanar: abstract class extends RasterPlanar {
 	}
 	distance: func (other: Image) -> Float {
 		result := 0.0f
-		if (!other)
+		if (!other || (this size != other size) || !other instanceOf?(This))
 			result = Float maximumValue
```

Comment:
Okay, so the reason we don't move this function into the non-abstract classes is to avoid the code duplication it would result in?

Anyway, I just noticed now that `distance` is 100% identical in this class and in `RasterYuvSemiplanar`. I guess that's better than 99%.